examples/consuming_packages/versioning/conflicts/override/run_example.py

```diff
@@ -0,0 +1,48 @@
+import os
+import subprocess
+
+def run(cmd, error=False):
+    # Used by tools/scm check_repo only (see if repo ok with status)
+    print("Running: {}".format(cmd))
+    process = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    out, err = process.communicate()
+    out = out.decode("utf-8")
+    err = err.decode("utf-8")
+    ret = process.returncode
+
+    output = err + out
+    if ret != 0 and not error:
+        raise Exception("Failed cmd: {}\n{}".format(cmd, output))
+    if ret == 0 and error:
+        raise Exception("Cmd succeded (failure expected): {}\n{}".format(cmd, output))
+    return output
+
+
+# This is a full diamond
+#                        game -> engine -> math/1.0
+#                          \----> ai -----> math/2.0 (conflict)
+# solved with override=True
+
+# Demo the conflict
+run("conan remove * -f")  # Make sure no packages from last run
+run("conan create math --version=1.0")
+run("conan create math --version=2.0")
+run("conan create engine")
+run("conan create ai")
+out = run("conan install game", error=True)
+# NOTE This output shows the downstream conflict not the immediate
+assert "ERROR: Version conflict: ai/1.0->math/2.0, game/1.0->math/1.0" in out
```

Comment:
that is weird. I would have expected now something like below:
```suggestion
assert "ERROR: Version conflict: ai/1.0->math/2.0, engine/1.0->math/1.0" in out
```
what am I missing?

Reply:
Nop, this is the current implementation. The conflict shown is because the engine->math conflict is propagated down to game. If it wasn't propagated it wouldn't conflict, so the conflict needs to be tracked down from game.

At least this is the current implementation, we can try to improve the information there, but that would require changes in code.

This will be run by CI, so we will make sure that the examples work and they align with the release.